### DISCUSSION TITLE
How to download on mac?

### DISCUSSION THREAD

**Discussion by kristiankg4** [apertium/organisation#35] [Q&A] [answered]
Hi, I am a Mac user and I want to download Apertium to use the "norsk bokmål - norsk nynorsk" translator. I think in english it may be called Norwegian - New norwegian. Is there any way to download it so it can be used offline? Thanks in advance :)

**Comment by mr-martian** [comment, marked as answer]
We have either a stand-alone GUI: https://wiki.apertium.org/wiki/Apertium_Simpleton_UI or a command-line interface: https://wiki.apertium.org/wiki/Apertium_on_Mac_OS_X
